Doc/library/logging.rst

```diff
@@ -254,9 +254,11 @@ is the module's name in the Python package namespace.
       Logs a message with level :const:`WARNING` on this logger. The arguments are
       interpreted as for :meth:`debug`.
 
-      .. note:: There is an obsolete method ``warn`` which is functionally
-         identical to ``warning``. As ``warn`` is deprecated, please do not use
-         it - use ``warning`` instead.
+      .. versionchanged:: 3.11
+
+      The deprecated ``logging.warn`` method, a duplicate of this method,
```

Comment:
Should this be ``logging.Logger.warn`` as its the method?

Reply:
Yes, good catch :) see d774806f29f3f19eade97632e8c2392573abf62c